tensorflow/core/kernels/data/dataset_utils.h

```diff
@@ -57,6 +57,58 @@ Status VerifyTypesMatch(const DataTypeVector& expected,
 Status VerifyShapesCompatible(const std::vector<PartialTensorShape>& expected,
                               const std::vector<PartialTensorShape>& received);
 
+constexpr char kDelimiter[] = "@@";
```

Comment:
this should move to an anonymous namespace in the .cc file, this will require defining the `VariantTensorDataReader` constructor out-of-line in the .cc file as well